content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,265 @@
+variety of options for consuming that data. The decision of a storage and
+analysis engine for telemetry has a variety of interplay with semantic
+conventions, and maintainers must consider the tradeoffs between cardinality,
+dimensionality, and specificity of attributes when designing conventions and
+stabilizing them.
+
+The downside of this is that progress on semantic conventions can be slow, and
+this slowness impacts all consumers of the conventions. Many instrumentation
+libraries are currently stuck on pre-release versions because they depend on
+experimental semantic conventions. Outside contributors are stuck between
+emitting unspecified telemetry or trying to engage in the process, which
+requires a long commitment. Finally, we're internally inconsistent in
+instrumentation across the project; Some libraries are mapped to conventions,
+others exist independently of it.
+
+### What's Changing?
+
+Our goals here are designed to achieve two outcomes.
+
+1. Semantic conventions should have more nuanced stability levels in order to
+   unblock upstream development and gain real world feedback from users.
+2. Semantic conventions should be more federated; We don't want the project to
+   be a constraint on the development and promulgation of the conventions.
+
+To this end, we have a few recommendations we'd like to codify into the
+specification. First, our position around instrumentation libraries in
+OpenTelemetry is that they exist as concrete implementations of the semantic
+conventions. This gives us a concrete target for 'first party' instrumentation
+libraries that we wish to support in distributions. In addition, maintainers
+should prioritize instrumentations that align to existing conventions and
+deprioritize others. Second, we would like to promote more first-party semantic
+conventions to 'beta' stability in order to unblock downstream instrumentation
+libraries. Many of our conventions have months, or years, of real-world use that
+we can rely on as guidance for their fitness. Finally, we'd like to formalize
+the semantic convention format specification to the point that third-parties
+outside the OpenTelemetry project feel comfortable using it to publish their
+own, federated, semantic conventions.
+
+In order to accomplish this, we're looking for feedback on several areas from
+maintainers and end-users -- especially around the maturity/lifecycle of
+semantic conventions, as well as what's missing in terms of federating semantic
+conventions. We are more flexible on proposals here, but our outcomes aren't.
+Remember, a core goal of the project is to encourage other libraries, tools, and
+frameworks to
+[natively adopt OpenTelemetry](https://www.youtube.com/watch?v=l8xiNOCIdLY) --
+semantic conventions are a big part of that.
+
+## 3. Confident, Stable, Releases.
+
+## Background
+
+OpenTelemetry isn't just one thing, especially in complex deployments. Subtle
```

Comment:
Suggest re-phrasing. "one thing" may reduce the importance of what is being conveyed. Maybe we could say "OpenTelemetry is not just one component being developed and released for complex deployments."